src/App.tsx

```diff
@@ -1,16 +1,21 @@
 import { BrowserRouter, Route, Routes } from "react-router-dom";
 import LoginPage from "./pages/login/login";
 import DashboardLayout from "./layout/dashboard-layout/dashboard-layout";
+import NotPage from "./pages/login/NotPage";
+import NotAcess from "./pages/login/NotAcess";
```

Comment:
NotPageni NotFound ga o'zgartiraylik,
Not Acess ni AccessDenied ga
routelarni ham shunga qarab nomlaymiz